CHANGELOG.md

```diff
@@ -1,3 +1,45 @@
+# 0.17.0
+## Overview
+
+This release brings a lot of changes and refactors among which the highlights are:
+* Refactor of JetStream API
```

Comment:
```suggestion
* A complete rewrite of the JetStream API with a new subscription interface
```

Reply:
Rewrite rather than refactor? As refactor kind of implies same level of functionality, we support a lot more now :)